Dockerfile

```diff
@@ -1,17 +1,17 @@
 # Container image that runs your code
 FROM docker.io/snyk/snyk:linux@sha256:91a21f9dfd09a93126d6b066f86f995013148253d98284f300a7b69c12e1829c as snyk
+# Note that the version of OPA used by pr-checks must be updated manually to reflect dependabot updates
```

Comment:
Maybe we can also add the example command to extract the version of the OPA from the updated conftest image, e.g.:
```
podman run --rm -ti ${NEW_IMAGE}  --version
```

Reply:
Ack good idea, updated in [aeedab0](https://github.com/redhat-appstudio/hacbs-test/commit/aeedab0dee8ca5fda7826ead3e8b660feb5affdd)